public/js/admin-scripts.js

```diff
@@ -9,3 +9,18 @@ li.each(function(){
 		});
 	};
 });
+
+
+   $(document).ready(function () {
+                $("#sidebar").mCustomScrollbar({
+                    theme: "minimal"
+                });
+
+				let sideBarStuff = $('#sidebar, #content')
```

Comment:
same thing for every selector.
    $('.collapse.in')
          $('a[aria-expanded=true]')